### DISCUSSION TITLE
Delegates are not installed on RHEL when we build and install image magick

### DISCUSSION THREAD

**Discussion by skalokhe** [ImageMagick/ImageMagick#6307] [Help]
### ImageMagick version

7.1.0.62

### Operating system, version and so on

RHEL8.3, 8,7

## Pre-req dependencies
sudo yum groupinstall "Development Tools" -y
yum -y install bzip2-devel freetype-devel libjpeg-devel libpng-devel libtiff-devel giflib-devel zlib-devel ghostscript-devel djvulibre-devel libwmf-devel jasper-devel libtool-ltdl-devel libX11-devel libXext-devel libXt-devel lcms-devel libxml2-devel librsvg2-devel OpenEXR-devel php-devel
## download imagemagic :
wget https://imagemagick.org/archive/releases/ImageMagick-7.1.0-62.tar.gz

## extact the tar ###
tar -xvf ImageMagick-7.1.0-62.tar.gz

cd ImageMagick-7.1.0-62
./configure --prefix=/usr
make
##If build fails, try gmake instead.
sudo make install
export LD_LIBRARY_PATH=/usr/lib
sudo ldconfig
ldconfig /usr/lib

#sudo ldconfig /usr/local/lib
#ls -l /usr/local/bin/magick
#cd ~
#echo "export PATH=$PATH:/usr/local/bin" >>.bashrc
#export PATH=$PATH:/usr/local/bin
convert --version

here we see only two delegated installed 
convert --version
Version: ImageMagick 7.1.0-62 Q16-HDRI x86_64 20885 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP(4.5)
Delegates (built-in): lzma zlib
Compiler: gcc (8.5)

i am not seeing the jpg, png, etc delegates getting installed.

**Comment by dlemstra** [comment]
Is it possible to reproduce this with a `Dockerfile` and can you share it? And do you have `pkg-config` installed? This might be the reason those delegates could not be found. And I would also advise you to check the `config.log` file it might tell you why it fails to find those delegates.
